scripts/deployment/globals_gnosis_mainnet.json

```diff
@@ -1 +1 @@
-{"contractVerification":true,"useLedger":true,"derivationPath":"m/44'/60'/2'/0/0","providerName":"gnosis","gasPriceInGwei":"5","baseURI":"https://gateway.autonolas.tech/ipfs/","agentRegistryName":"AI Agent Registry","agentRegistrySymbol":"AI-AGENT-V1","agentRegistryAddress":"0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA","agentFactoryAddress":"0x88DE734655184a09B70700aE4F72364d1ad23728"}
\ No newline at end of file
+{"contractVerification":true,"useLedger":true,"derivationPath":"m/44'/60'/2'/0/0","providerName":"gnosis","gasPriceInGwei":"5","baseURI":"https://gateway.autonolas.tech/ipfs/","agentRegistryName":"AI Agent Registry","agentRegistrySymbol":"AI-AGENT-V1","agentRegistryAddress":"0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA","agentFactoryAddress":"0x88DE734655184a09B70700aE4F72364d1ad23728","agentId":"1","price":"1000000000000000000"}
```

Comment:
`agentId` and `price` are given for mech verification purposes only (`e_check_04_agent_mech.js`).